google/cloud/webrisk/README.md

```diff
@@ -25,7 +26,7 @@ Please note that the Google Cloud C++ client libraries do **not** follow
   client library
 * Detailed header comments in our [public `.h`][source-link] files
 
-[cloud-service-docs]: https://cloud.google.com/webrisk
+[cloud-service-docs]: https://cloud.google.com/web-risk
```

Comment:
this is unused, because the link was inlined.

Reply:
Fixed, thanks.